### ISSUE TITLE
Add description for step

### ISSUE THREAD

**Issue by huangkai31** [orchest/orchest#775]
**Describe the feature you'd love to have in Orchest**
Would love to add a description input box , support markdown  in step properties
![image](https://user-images.githubusercontent.com/1758277/157361498-3e62210e-8f96-4959-852c-6216eb064900.png)

**Comment by yannickperrenet** [comment]
I love the suggestion. Would you be okay with the description only showing in the properties pane? Meaning that you won't be able to see a step's description unless you open its properties.

**Comment by huangkai31** [comment]
Yes, for description mainly for explain the title, so just properties pane is good enough.

**Comment by ricklamers** [comment]
It would be great if the description supports Markdown
